### PR TITLE
Sentence transformers

### PR DESCRIPTION
**Pull request by abhijithneilabraham**
Support for #760.

Sample Usage: 
```
from flash.text.embeddings import SentenceEmbeddings

sentence_embeddings=SentenceEmbeddings('all-MiniLM-L6-v2')

s=["I am good","how are you"]
print(sentence_embeddings.generate_embeddings(s))
```

@SeanNaren Can you review this and tell me if I am in the right direction? If yes I can write the fine-tuning API as well.

### CODE REVIEW

File: flash/text/embeddings/sentence_embeddings.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+from sentence_transformers import SentenceTransformer
+from torch import Tensor
+
+
+class SentenceEmbeddings:
+    def __init__(self, model_name: str):
+        self.model_name = model_name
+        self.model = SentenceTransformer(model_name_or_path=self.model_name)
+
+    def return_embeddings(
```
Comment (rohitgr7) [inline flash/text/embeddings/sentence_embeddings.py:27]: how about `generate_embeddings`?

Comment (abhijithneilabraham) [reply flash/text/embeddings/sentence_embeddings.py]: @rohitgr7 Sure. Fixed that.

**Comment by tchaton** [comment]
Hey @abhijithneilabraham,

Thanks for your contribution. I am bit a perplex with this integration.
It is relying entirely on sentence-transformers to provide embeddings. I wonder how it would play with the different datamodule we currently support.

Does the user have any way to train the embeddings ?

Best,
T.C

**Comment by abhijithneilabraham** [comment]
Hi,
Can you guide me to the datamodule you support? I'll start working from
there.

**Comment by SeanNaren** [comment]
hey @abhijithneilabraham I appreciate the very minimal integration, its good to start off from somewhere :)

your best bet is to follow the documentation guide on setting up a task: https://lightning-flash.readthedocs.io/en/latest/template/intro.html

It's important to follow the guide and really leverage the base classes Flash has to offer to ensure that we can support a large suite of features! Let me know how this goes :)

**Comment by abhijithneilabraham** [comment]
Support for #760.

Sample Usage: 
```
from flash.text.embeddings import SentenceEmbedder

sentence_embedder=SentenceEmbedder()
print(sentence_embedder.generate_embeddings(["Today was a good day"]))
```

@SeanNaren , Hi, would this be a good start to creating the sentence embedder task? Let me know if I am in the right direction.

**Pull request closed without merging**